include/tkey/led.h

```diff
@@ -17,6 +17,7 @@
 #define LED_WHITE (LED_RED | LED_GREEN | LED_BLUE)
 // clang-format on
 
-void set_led(uint32_t led_value);
-void forever_redflash();
+uint32_t getled();
+void setled(uint32_t ledvalue);
+void flashforever(uint32_t ledvalue);
```

Comment:
What's up with the new naming convention on some variables/functions?
Isn't a better convention to first describe the "module" it comes from and then the action of the function?
`led_set()`, `led_get()` etc.? Same goes for `led_value` instead of `ledvalue`.
As you do with for example `timer_ctrl` etc, think the mixing of styles makes it look a bit unstructured..

I think readability suffers when using `setled()` and so on. If we would go this way, why not then start using CamelCase?
Only writing one comment about it, but It goes for all variables named like it.